app/src/androidTest/kotlin/cz/covid19cz/erouska/screenObject/PhoneNumberScreen.kt

```diff
@@ -0,0 +1,28 @@
+package cz.covid19cz.erouska.screenObject
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import cz.covid19cz.erouska.R
+
+class PhoneNumberScreen {
+    companion object {
+        const val PHONE_NUMBER = "731 000 000"
+    }
+
+    fun typePhoneNumber() {
+        onView(withId(R.id.login_desc)).check(matches(isDisplayed()))
+        onView(withId(R.id.login_verif_phone_input)).perform(typeText(PHONE_NUMBER.replace(" ", "")), closeSoftKeyboard())
```

Comment:
Do políčka by mělo jít vložit i číslo s mezerama

Reply:
Zkusil jsem 
`onView(withId(R.id.login_verif_phone_input)).perform(replaceText("+420$PHONE_NUMBER"), closeSoftKeyboard())` Ale text se nevejde do inputu

<img width="366" alt="Snímek obrazovky 2020-04-17 v 0 30 54" src="https://user-images.githubusercontent.com/230124/79512924-d3eddf00-8042-11ea-9338-c3d680e521eb.png">

Reply:
Máš určitě nejnovější develop? Před asi 5 dny jsme zvedali počet znaků v tom inputu.